br/tests/rawkv/run.py

```diff
@@ -16,44 +30,76 @@ def __init__(self, global_args):
         self.br_storage = global_args.br_storage
 
 
-    def test_rawkv(self):
+    def test_dst_apiv1(self):
+        self._clean()
+        self._run_br_test("v1")
+        self._success_msg(self.test_dst_apiv1.__name__)
+
+
+    def test_dst_apiv1ttl(self):
+        self._clean()
+        self._run_br_test("v1ttl")
+        self._success_msg(self.test_dst_apiv1ttl.__name__)
+
+
+    def test_dst_apiv2(self):
+        self._clean()
+        self._run_br_test("v2")
+        self._success_msg(self.test_dst_apiv2.__name__)
+
+    def _clean(self):
+        if self.br_storage.startswith("local://"):
+            local_dir = self.br_storage[len("local://"):]
+            self._run_cmd("rm", "-rf", local_dir)
```

Comment:
`rm -rf` would be dangerous, in case someone accidently set an important path (e.g. `/`).

I think we can use the function names as sub directories, to make the storage path of a single test clean.

Reply:
done